app/js/app/directives/EventBookingForm.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2015 Stephen Cummins
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([], function() {
+
+
+	return ["api", "$rootScope", function(api, $rootScope) {
+
+		return {
+			scope: true,
+
+ 			restrict: "A",
+			
+			templateUrl: "/partials/event_booking_form.html",
+
+			link: function(scope, element, attrs) {
+
+				scope.editingSelf = scope.targetUser._id == scope.user._id
```

Comment:
I get lots of warnings about `scope.targetUser` being `null` and so not being able to have a property `_id`. Since the line gets run when the form gets added to the DOM at pageload time, it's always incorrect anyway. It's not as issue because the method isn't used. See next comment.